src/translations/fr/case-studies.mdx

```diff
@@ -1,187 +1,49 @@
 
-Que **Pinterest** avait retiré leurs mentions au Privacy Shield dans leurs conditions d’utilisation respectives. Dans un arrêt rendu jeudi 16 juillet 2020 (Arrêt Schrems II), la Cour de Justice de l’Union Européenne (CJUE) a en effet invalidé ce texte, qui venait encadrer les transferts de données entre l’Union européenne et les Etats-Unis.
+Dans la mise à jour des ses conditions d’utilisation, Instagram interdit de créer des comptes ou de les utiliser de sorte à récupérer des informations sans autorisation, en particulier de manière automatisée. Ceci pourrait avoir un effet sur le travail de chercheurs comme ceux de l’ONG Algorithm Watch : un projet de recherche de cette dernière s’était déjà retrouvé fermé par Instagram <a  target="_blank" rel="noopener" href="https://algorithmwatch.org/en/instagram-research-shut-down-by-facebook/">à l’été 2021</a>. La plateforme précise aussi l’interdiction de diffuser les informations personnelles d’un autre utilisateur, y compris des données ou des publications relevant de la propriété intellectuelle d’un ou une autre internaute.
 
-<LinkIcon
-  iconColor="var(--colorBlack400)"
-  href="https://github.com/ambanum/OpenTermsArchive-versions/commit/f3b4d04b4"
-  small={true}
-  className="mb__L"
->
-  <a target="_blank" rel="noopener">
-    Voir le changement
-  </a>
-</LinkIcon>
+Enfin, le réseau social développe le type de données qu’il collecte dans ses politiques de confidentialité : des métadonnées sur les outils utilisés par l’internaute, ce que l’internaute voit ou enregistre avec sa caméra, les publications et les comptes avec lesquels il interagit ou encore la fréquence, la durée et l’heure à laquelle il ou elle réalise ces activités. Les partenaires de Meta reçoivent tout ou partie des données lorsque leurs services sont utilisés.
 
 ---
 
-### Snapchat
-
-#### Le 22 septembre 2020
-
-Que **Snapchat** avait retiré leurs mentions au Privacy Shield dans leurs conditions d’utilisation respectives. Dans un arrêt rendu jeudi 16 juillet 2020 (Arrêt Schrems II), la Cour de Justice de l’Union Européenne (CJUE) a en effet invalidé ce texte, qui venait encadrer les transferts de données entre l’Union européenne et les Etats-Unis.
-
-<LinkIcon
-  iconColor="var(--colorBlack400)"
-  href="https://github.com/ambanum/OpenTermsArchive-versions/commit/27beddd06"
-  small={true}
-  className="mb__L"
->
-  <a target="_blank" rel="noopener">
-    Voir le changement
+<div className="text__center">
+  D’autres cas d’usage sont accessibles{' '}
+  <a
+    target="_blank"
+    rel="noopener"
+    href="https://github.com/ambanum/OpenTermsArchive/wiki/Études-de-cas"
+  >
+    à ce lien
   </a>
-</LinkIcon>
-
----
-
-D’autres cas d’usage sont accessibles <a href="https://github.com/ambanum/OpenTermsArchive/wiki/Études-de-cas" target="_blank" rel="noopener">à ce lien</a>.
+  .
+</div>
```

Comment:
Let's get rid of this untranslated, not well presented link. We should at some point incorporate more of these examples into this page and erase the wiki page 🙂

Reply:
done

Reply:
I deleted the wiki page.